app/src/components/register/DropZone.module.scss

```diff
@@ -0,0 +1,228 @@
+  -o-transition: .3s ease;
+  transition: .3s ease;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 40%;
+    left: 60%;
+    -webkit-transform: translate(-50%, -50%);
+    -moz-transform: translate(-50%, -50%);
+    -o-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    width: 40%;
+    height: 40%;
+    border: dashed 2px rgba(255, 255, 255, .9);
+    border-radius: 5px;
+    -webkit-transition: .3s ease;
+    -moz-transition: .3s ease;
+    -o-transition: .3s ease;
+    transition: .3s ease;
+  }
+
+  &:after {
+    content: '';
+    position: absolute;
+    top: 60%;
+    left: 40%;
+    -webkit-transform: translate(-50%, -50%);
+    -moz-transform: translate(-50%, -50%);
+    -o-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    width: 45%;
+    height: 45%;
+    border-radius: 5px;
+    background: rgba(255, 255, 255, .3);
+    -webkit-transition: .3s ease;
+    -moz-transition: .3s ease;
+    -o-transition: .3s ease;
+    transition: .3s ease;
+  }
+}
+
+.progress {
+  position: absolute;
+  z-index: 0;
+  top: 0;
+  left: 0;
+  width: 0;
+  height: 100%;
+  background: #fff;
+  -webkit-transition: 1s ease;
+  -moz-transition: 1s ease;
+  -o-transition: 1s ease;
+  transition: 1s ease;
+}
+
+.progress.complete {
+  width: 100%;
+}
+
+.dropzone.fileOver .box {
+  width: 80px;
+  height: 3px;
+  border-radius: 100px;
+  top: 70%;
+}
+
+.dropzone.fileOver .box:before, .dropzone.fileOver .box:after {
+  -webkit-transform: translate(-50%, -50%) scale(0);
+  -moz-transform: translate(-50%, -50%) scale(0);
+  -o-transform: translate(-50%, -50%) scale(0);
+  -ms-transform: translate(-50%, -50%) scale(0);
+  transform: translate(-50%, -50%) scale(0);
+}
+
+/* Arrow icon */
+.arrow {
+  opacity: 0;
+  position: absolute;
+  z-index: 0;
+  top: 30%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -moz-transform: translate(-50%, -50%);
+  -o-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  width: 35px;
+  height: 35px;
+  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/269193/arrow-down.png) no-repeat center;
+  background-size: 60px;
+  -webkit-transition: .3s ease;
+  -moz-transition: .3s ease;
+  -o-transition: .3s ease;
+  transition: .3s ease;
+  overflow: hidden;
+}
+
+/* Popup */
+.top {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 500px;
+}
+
+.popup {
+  position: absolute;
+  top: 6%;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  -moz-transform: translateX(-50%);
+  -o-transform: translateX(-50%);
+  -ms-transform: translateX(-50%);
+  transform: translateX(-50%);
+  width: 300px;
+  height: auto;
+  padding: 20px 15px;
+  box-shadow: 0 3px 20px 2px rgba(0, 0, 0, .1);
+  border-radius: 5px;
+  z-index: 10;
+}
+
+.profile-avatar {
```

Comment:
Assuming this is a set name you have to use? If not would be nice to rename since profile avatar (in the scope of all of our code) seems to me like it would refer to the user account's avatar/image before it would refer to this